CryptoLightNewsTests/CryptoLightNewsTests.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CryptoLightNewsTests.swift
+//  CryptoLightNewsTests
+//
+//  Created by Richard Price on 02/01/2023.
+//
+
+import XCTest
+
+@testable import CryptoLightNews
+
+
+final class CryptoLightNewsTests: XCTestCase {
+    
+    var viewController: NewsViewController!
+    // var mockAPIService: DataFetcherService!
+    
+    override func setUp() {
+        super.setUp()
+        viewController = NewsViewController()
+        //mockAPIService = DataFetcherService(networkService: Service.init())
+        //viewController.service = mockAPIService
+        viewController.loadViewIfNeeded()
+    }
+    
+    override func tearDown() {
+        viewController = nil
+        //mockAPIService = nil
+        super.setUp()
+    }
+    
+    func testItemPropertyTitleWhenSet() {
+        let cell = CustomNewsControllerCell(frame: .zero)
+        let item = NewsArticles(title: "test title")
+        cell.item = item
+        XCTAssertEqual(cell.item?.title, item.title)
+    }
+    
+    func testArticleLabel() {
+        let cell = CustomNewsControllerCell()
```

Comment:
Does this test belong here? 🤔 Same as above... it seems that you have a justification to create another test file 😄